app/controllers/admin/billboards_controller.rb

```diff
@@ -17,6 +17,7 @@ def new
 
     def edit
       @billboard = Billboard.find(params[:id])
+      @test_article = Article.last
```

Comment:
This should only include published articles, so `Article.published.last` should work.